python-avd/pyavd/api/interface_descriptions/__init__.py

```diff
@@ -261,6 +261,14 @@ def vtep_loopback_interface(self, data: InterfaceDescriptionData) -> str:
 
         return data.description
 
+    def wan_ha_ethernet_interface(self, data: InterfaceDescriptionData) -> str:
+        """Build WAN HA ethernet interface description."""
+        return f"WAN_HA_{data.peer}_{data.peer_interface}"
+
```

Comment:
Please add the available data fields in the docstring for someone overloading this.